editor/src/clj/editor/cljfx_form_view.clj

```diff
@@ -1288,7 +1288,7 @@
                        (remove :hidden?)
                        (map #(set-field-visibility % values filter-term visible)))
                      fields)]
-    (assoc section :visible (or visible (boolean (some :visible fields)))
+    (assoc section :visible (boolean (some :visible fields))
```

Comment:
Sections with nothing but deprecated fields were showing up as empty pages in the Form View. The `section-visibility` is already considered when filtering the fields, so we should not involve it here as well.